SwedbankPaySDKTests/GoodWebViewRedirectsTests.swift

```diff
@@ -0,0 +1,93 @@
+import XCTest
+import Foundation
+@testable import SwedbankPaySDK
+
+class GoodWebViewRedirectsTests: XCTestCase {
+    private var redirects: GoodWebViewRedirects!
+    
```

Comment:
Codacy found an issue: [Lines should not have trailing whitespace.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6137886)